src/sentry/static/sentry/app/redux/actions/shared.js

```diff
@@ -9,3 +11,95 @@ export function ac(type, ...argNames) {
     return action;
   };
 }
+
+// Action creators for standard actions
+
+// List actions:
+export const acRequest = resource => {
+  return (search = null, groupBy = null, cursor = null) => {
+    return {
+      type: `GET_${resource}_LIST_REQUEST`,
+      search,
+      groupBy,
+      cursor,
+    };
+  };
+};
+
+export const acSuccess = resource => {
+  return (entries, link) => {
+    return {
+      type: `GET_${resource}_LIST_SUCCESS`,
+      entries,
+      link,
+    };
+  };
+};
+
+export const acFailure = resource => {
+  return err => ({
+    type: `GET_${resource}_LIST_FAILURE`,
+    message: err,
+  });
+};
+
+export const acGetList = (resource, url) => {
+  return (search, groupBy, cursor) => dispatch => {
+    dispatch(acRequest(resource)(search, groupBy, cursor));
+
+    const config = {
+      params: {
+        search,
+        cursor,
+      },
+    };
+
+    return axios
+      .get(url, config)
+      .then(res => dispatch(acSuccess(resource)(res.data, res.headers.link)))
+      .catch(err => dispatch(acFailure(resource)(err)));
+  };
+};
+
+export const acSelectPage = resource => {
+  return doSelect => {
+    return {
+      type: `TOGGLE_SELECT_PAGE_OF_${resource}`,
+      doSelect,
+    };
+  };
+};
+
+export const acSelect = resource => {
+  return (id, doSelect) => {
+    return {
+      type: `TOGGLE_SELECT_${resource}`,
+      id,
+      doSelect,
+    };
+  };
+};
+
+export const list = {
```

Comment:
Perhaps "actionCreators"

Reply:
Ahh, you mean `listActionCreators` or? Because the idea is that these are generators for `list` actions. The idea is not that this is the list of all action creators.

But since I needed to explain that, perhaps `listActionCreators` would be better.